docs/submission/clinical-data-validation-rules.md

```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
 
 ![Reminder Banner](/assets/submission/banner-reminder.svg)
 
+## Clinical Data Encoding Rules
+
+### Identifier fields
+
+The data dictionary contains certain data elements regarded as "identifiers". These fields have an ![ID](/assets/submission/dictionary-id.svg) descriptor in the data dictionary and include:
+
+- Donor: `submitter_donor_id`
+- Specimen: `submitter_specimen_id`
+- Sample: `submitter_sample_id` 
+- Primary Diagnosis: `submitter_primary_diagnosis_id`
+- Treatment: `submitter_treatment_id`
+- Follow Up: `submitter_follow_up_id`
+
+These fields must to be coded specifically for ICGC-ARGO purposes using the following rules:
+
+- These identifiers should not be derived from IDs such as biobank or hospital identifiers or any other personal identifying information. These IDs are to be coded in such a way that they cannot be tracked back to the individual donors, except by the submitting program. Only the program will keep the key that will permit the data to be linked back to the individual donors. This key must not be communicated to the data users.
```

Comment:
suggestion but please ignore if wrong..

These identifiers should not be derived from biobank or hospital identifiers or any other personal identifying information.